pkg/oceanbase/const/sql/cluster.go

```diff
@@ -13,6 +13,6 @@ See the Mulan PSL v2 for more details.
 package sql
 
 const (
-	BootstrapServer = "REGION '%s' ZONE '%s' SERVER '%s:%d'"
```

Comment:
Why delete REGION here?

Reply:
region is not required, and it can be set after the cluster starts